iron-overlay-behavior.html

```diff
@@ -259,7 +259,7 @@
 
       // Defer any animation-related code on attached
       // (_openedChanged gets called again on attached).
-      if (!this.isAttached) {
+      if (this.isAttached === undefined) {
```

Comment:
This would cause executing `__openedChanged` (and trigger relayouts) also when the overlay is detached.
The fix should be done in `detached`, where we need to check if the overlay `__isAnimating` and call `_applyFocus()` if so. Addressing this in https://github.com/PolymerElements/iron-overlay-behavior/pull/230